### DISCUSSION TITLE
Natural gas price

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#115] [Q&A: Data]
Hi ReEDS team,

I have three questions about the natural gas price and supply curves in ReEDS.

(1) What guidance should I use when choosing Sw_GasCurve? For example, the b_input.gms  has guidance that "If you are only running ERCOT, Sw_GasCurve needs to be 2". Is there any other guidance on the choice for other Sw_GasCurve cases?

(2) How is the census division multiplier calculated? In the ReEDS 2021 technical documentation, it says "With a national-only supply curve, there is a census division multiplier to adjust the price for each census division".

(3) In the b_input.gms file (attached screenshot), what does the 0.5 in the gasquant(cendiv,gb,t) code stand for?
![image](https://user-images.githubusercontent.com/78582696/164537310-a013518b-b0a1-4084-b6c5-b49097f8f4e1.png)

Thank you!

**Comment by vcelloho** [comment]
1) The possible settings for natural gas supply curves is 0 for census division, 1 national supply curve + census division, 2 static fixed price trajectory, 3 national supply curve only. My understanding of the reason for this is that ERCOT is a subset of the West South Central census division. This could create odd behavior with the prices and quantities of natural gas consumed, hence the need for running with static prices.

2) Natural gas prices are calculated in a method documented here. We update the corresponding national and regional parameters as each AEO is released.
![image](https://user-images.githubusercontent.com/2414286/164542802-69897ace-d8e9-4fd3-9646-ee8ac9a6c938.png)
https://www.sciencedirect.com/science/article/pii/S0140988316300433 

3) These equations are used for dividing up the natural gas supply curve quantities into the specified number of equally distributed bins.

**Reply by wanyingw193** [reply]
Thank you! It is clearer now.
